libs/mep/mwpm-143159/timeline/timeline.css

```diff
@@ -0,0 +1,142 @@
+.timeline {
+  box-sizing: border-box;
+  padding: 0 0 var(--spacing-xl) 0;
+  width: var(--grid-container-width);
+  max-width: 968px;
+  margin: 0 auto;
+}
+
+.timeline .row {
+  display: grid;
+}
+
+.timeline h1,
+.timeline h2,
+.timeline h3,
+.timeline h4,
+.timeline h5,
+.timeline h6,
+.timeline p {
+  color: var(--text-color);
+  margin: 0;
+  font-size: var(--type-body-xs-size);
+  line-height: var(--type-body-xs-lh);
+}
+
+.timeline .row:nth-child(2)>div {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.timeline .row:nth-of-type(1)>.right>div+div {
+  text-align: right;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.timeline .row:nth-of-type(1)>.left>div>* {
+  max-width: 100px;
+  padding-right: var(--spacing-m);
+}
+
+.dark .timeline .row:nth-child(3)>div *,
+.timeline.dark .row:nth-child(3)>div * {
+  color: var(--color-black);
+}
+
+.timeline .row:nth-of-type(1)>.right>div+div>* {
+  max-width: 100px;
+}
+
+.timeline .row:nth-of-type(1)>.right,
+.timeline .row:nth-of-type(2)>.right {
+  display: grid;
+  justify-content: space-between;
+}
+
+.timeline .row:nth-of-type(1)>.right {
+  grid-template-columns: minmax(110px, 1fr) 1fr;
+}
+
+.timeline .row:nth-of-type(2)>.right {
+  grid-template-columns: 2px 2px;
+}
+
+.timeline .row:nth-of-type(1)>.right>div:nth-of-type(1) {
+  max-width: 135px;
+  padding-right: var(--spacing-xxs);
+}
+
+.timeline .bar {
+  height: 20px;
+  width: 2px;
+  border-radius: 1px;
+  margin: var(--spacing-xxs) 0;
+}
+
+.timeline .row:nth-of-type(3) p {
+  text-align: center;
+  font-weight: 700;
+  padding: 4px var(--spacing-xxs);
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 4px;
+}
+
+.timeline .row:nth-of-type(3) .left p {
+  margin-right: 1px;
+}
+
+.timeline .row:nth-of-type(3) .right p {
+  margin-left: 3px;
+}
+
+.dark .timeline .row:nth-child(1)>div>div>*,
```

Comment:
ok, so this is where you're forcing the bg color to be white.  So is the goal to never allow a dark variant version of this block?

Reply:
see comment above.